packages/authorization-updater/src/authorizationService.ts

```diff
@@ -142,5 +160,39 @@ export const authorizationServiceBuilder = (
         `Updated Agreement ${agreementId} and EService ${eserviceId} states for all clients`
       );
     },
+    async deletePurposeFromClient(
+      purposeId: PurposeId,
+      clientId: ClientId,
+      logger: Logger,
+      correlationId: string
+    ) {
+      const token = (await refreshableToken.get()).serialized;
+      const headers = getHeaders(correlationId, token);
+      await authMgmtClient.removeClientPurpose(undefined, {
+        params: { purposeId, clientId },
+        withCredentials: true,
+        headers,
+      });
+      logger.info(`Deleting purpose ${purposeId} from client ${clientId}`);
```

Comment:
For consistency, we could align these logs with the ones of the other methods in the file: there I used the past tense because the log is after the call ("Updated") - feel free to align and use the -ing or the past everywhere